### PR TITLE
Update sponsors section

### PR DESCRIPTION
**Pull request by dunja-tomic**
Moved last year's sponsors to "Our Previous Sponsors" and added the logos for this year's sponsors

### CODE REVIEW

File: src/components/Sponsors/index.js [hunk 1]
```diff
@@ -1,94 +1,115 @@
...
+import jtdc_logo from "../../assets/sponsor_logos_2020/jtdc_logo.png";
+import shopify_logo from "../../assets/sponsor_logos_2020/shopify_logo.png";
+import sunlife_logo from "../../assets/sponsor_logos_2020/sunlife_logo.jpg";
+import uphabit_logo from "../../assets/sponsor_logos_2020/uphabit_logo.png";
+import voiceflow_logo from "../../assets/sponsor_logos_2020/voiceflow_logo.png";
 
 import styles from "./styles.module.css";
 
-export default class Sponsors extends Component {
+export default class PreviousSponsors extends Component {
```
Comment (vipj) [inline src/components/Sponsors/index.js:15]: I think this is supposed to be "class Sponsors" and not "class PreviousSponsors"

**Review by vipj — APPROVED** [review]
Looks good! Just left a comment :)

**Review by tailaiwang — APPROVED** [review]
LGTM

File: src/components/Sponsors/index.js (hunk 1 above)
Comment (dunja-tomic) [reply src/components/Sponsors/index.js]: Good catch!

**Pull request merged**